license.txt

```diff
@@ -1,6 +1,7 @@
-Copyright 2022 diwako
+Original Copyright 2020-2022 diwako
+Adaptions 2022 Snick0rz/Celene 
 
-This work (diwako_cbrn or the like) uses the license Arma Public License Share Alike (APL-SA)
+This work is a derivitive of the work diwako_cbrn by diwako that uses the license Arma Public License Share Alike (APL-SA); as per the license, this work is also licensed under the APL-SA license.
```

Comment:
no need for this anymore :D

Reply:
btw you can resolve done suggestions with the "Resolve conversation" button below. Makes it easier to track open things.